addon/mixins/osf-model.js

```diff
@@ -7,5 +7,37 @@ import DS from 'ember-data';
 
 export default Ember.Mixin.create({
     links: DS.attr('links'),
-    embeds: DS.attr('embed')
+    embeds: DS.attr('embed'),
+
+    relationshipLinks: DS.attr('links'),
+    dirtyRelationships: Ember.computed('_dirtyRelationships', function() {
+        var dirtyRelationships = this.get('_dirtyRelationships');
+        return Object.keys(dirtyRelationships).map((rel) => {
+            if (rel === 'files' || rel === 'node' || rel === 'user') {
+                return null;
+            }
+            if (dirtyRelationships[rel]) {
+                return rel;
+            }
+            return null;
+        }).filter(Boolean);
+    }),
+    _dirtyRelationships: {},
+    onLoad: Ember.on('ready', function() {
```

Comment:
Not a big deal, but instead of adding an observer, can you just add a function named `didLoad` to use existing events?  Eg
http://jsfiddle.net/pangratz666/Vx8yL/

(Honestly asking. But if we can use an existing method hook, that's preferable)